test/http_server_spec/server.rb

```diff
@@ -30,7 +30,7 @@
 end
 
 get "/simple_get_with_body" do
-  "Hello world"
+  request.body
```

Comment:
Is this change necessary?  I think for as much as you can, do not touch the `http_server_spec` code.  Only remove the `@wip`-s as you get stories to pass.

Reply:
This was mistakenly left in from when I was worried that the acceptance tests were preventing the test from passing! I'll change that back right now.